script/Ciel/Day 14/14_01A_CIEL14A_5.txt

```diff
@@ -746,42 +807,45 @@ A night where the two of us were strolling around the city together.
 [sha:b2b631fbe105621487da311332e815f21373115a]{
 -- Page 203, Offset 38230.
 -- 口ゲンカばかりで、互いに触れる事はなかったけれど。
-Although we were constantly fighting and never came into contact with each other.
+Even if we did spend the whole time arguing with each other,
+ and never actually touched.
 }
 [sha:55279b61567d2df237a1a3714b5d48099ee0b7ec]{
 -- Page 204, Offset 38231.
 -- 「うん、でも―――
-"Yeah. But―――
+"Yeah... But you know what―――
 }
 [sha:a39b19f94580a4ccfca981836776163304735c84]{
 -- Page 204, Offset 38232.
 -- 志貴の、そういうところが好きだったなあ」
-That's what I like about you."
+I actually kind of liked that about you."
 }
 [sha:622d6566af10d03e45c83a28c2cabbc04436d5a2]{
 -- Page 205, Offset 38233.
 -- そう言って。
-As she says that...
+As she says that―――
```

Comment:
I think we can get something more out of the structure here (since the intent is emphatic quality)
>Leaving those final words behind―――